ui/packages/consul-ui/app/components/menu-panel/skin.scss

```diff
@@ -32,13 +25,6 @@
   border-color: rgb(var(--tone-gray-300));
   background-color: rgb(var(--tone-gray-000));
 }
-%menu-panel dt,
-%menu-panel dd {
-  color: rgb(var(--tone-gray-800));
-}
-%menu-panel dt span {
-  color: rgb(var(--tone-gray-600));
-}
 %menu-panel-separator {
   color: rgb(var(--tone-gray-600));
 }
```

Comment:
Most of the stuff I picked out and moved to its own .auth-profile/AuthProfile component is in red here.